frontend/lib/ui/Badge/Badge.module.css

```diff
@@ -60,7 +60,7 @@
   color: var(--badge-extra-entry);
 }
 
-.not_started,
+.first_entry_not_started,
 .first_entry_in_progress,
 .first_entry_unfinished {
```

Comment:
This can be removed

Reply:
Done in [d3c3073](https://github.com/kiesraad/abacus/pull/750/commits/d3c3073f4899e16ff8df3bf6b230b2ed19b34cef)

Reply:
Fully fixed in fea0fec47af69848f22a0744166a1cb99d35241d